certified-connectors/Oneflow/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1897 @@
+              "format": "binary"
+            }
+          },
+          "302": {
+            "description": "302",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "400": {
+            "description": "400",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "404": {
+            "description": "404",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "409": {
+            "description": "409",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/contracts/{contract_id}/": {
+      "delete": {
+        "summary": "Delete a contract",
+        "description": "Delete a specific contract by its ID.",
+        "operationId": "DeleteAContractById",
+        "parameters": [
+          {
+            "name": "contract_id",
+            "default": "10015",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "Contract Id",
+            "description":"Contract Id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object"
+            }
+          },
+          "400": {
+            "description": "400",
+            "schema": {
+              "type": "object"
+            }
+          },
+          "401": {
+            "description": "401",
+            "schema": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/contracts/{contract_id}": {
+      "get": {
+        "summary": "Get the status of a contract",
+        "description": "Retrieve the status of a specific contract by its ID.",
+        "operationId": "GetContractStatus",
+        "parameters": [
+          {
+            "name": "contract_id",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "Contract Id",
+            "description":"Contract Id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "state": {
+                  "type": "string",
+                  "description": "draft/pending/overdue/signed/declined",
+                  "x-ms-summary": "Contract state"
+                },
+                "state_updated_time": {
+                  "type": "string",
+                  "x-ms-summary": "Contract state update time"
+                },
+                "updated_time": {
+                  "type": "string",
+                  "x-ms-summary": "Contract update time"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "400",
```

Comment:
@DmitryKleshchov Can you please provide valid response code descriptions instead of just providing the response codes.

Applies the same comment for all the properties in the swagger.

Reply:
@sriyen-msft  done.